### PR TITLE
Feat: get workspace user information

### PR DESCRIPTION
**Pull request by evillalba94**
## Updates
- Allow searching for `limit` and `usage` belonging to `workspaceUser`.

## PR Related
- SDK: https://github.com/internxt/sdk/pull/216

### CODE REVIEW

**Comment by evillalba94** [comment]
@sg-gs Could you review this? Lmk

File: src/app/routes/bridge.ts [hunk 1]
```diff
@@ -13,19 +14,41 @@ class BridgeController {
   }
 
   async getUsage(req: Request, res: Response) {
-    const usage = await this.service.User.getUsage((req as AuthorizedRequest).user);
+    let user = (req as AuthorizedRequest).user;
+    const { workspaceUserId } = req.query;
```
Comment (sg-gs) [inline src/app/routes/bridge.ts:18]: Do not destructure if this param is optional, the code could crash on execution time. When getting information from an endpoint do this instead:

`const workspaceUserId = req.query.workspaceUserId`

File: src/app/routes/bridge.ts [hunk 2]
```diff
@@ -13,19 +14,41 @@ class BridgeController {
...
+
+    if (workspaceUserId) {
+      user = await this.service.User.FindUserByUuid(workspaceUserId).catch(() => {
```
Comment (sg-gs) [inline src/app/routes/bridge.ts:21]: This code could be cleaner doing this: 

```
const user = await this.service.User.FindUserByUuid(workspaceUserId)

if (!user) {
  throw new createHttpError(404, 'Workspace user not found');
}
```

File: src/app/routes/bridge.ts [hunk 3]
```diff
@@ -13,19 +14,41 @@ class BridgeController {
...
   async getLimit(req: Request, res: Response) {
-    const { 
+    let { 
       bridgeUser, 
       userId: bridgePass,
       updatedAt,
       uuid
     } = (req as AuthorizedRequest).user;
 
+    const { workspaceUserId } = req.query;
```
Comment (sg-gs) [inline src/app/routes/bridge.ts:39]: Same here with the destructuring

File: src/app/routes/bridge.ts [hunk 4]
```diff
@@ -13,19 +14,41 @@ class BridgeController {
...
+
+    if (workspaceUserId) {
+      const workspaceUser = await this.service.User.FindUserByUuid(workspaceUserId).catch(() => {
```
Comment (sg-gs) [inline src/app/routes/bridge.ts:42]: Same here with making the code easier to clean

**Review by sg-gs — CHANGES_REQUESTED** [review]

**Review by sg-gs — APPROVED** [review]

**Pull request closed without merging**